modules/lightning_features/lightning_media/src/Plugin/Field/FieldWidget/EntityReferenceBrowserWidget.php

```diff
@@ -1,41 +0,0 @@
-<?php
-
-namespace Drupal\lightning_media\Plugin\Field\FieldWidget;
-
-use Drupal\Core\Field\FieldItemListInterface;
-use Drupal\Core\Form\FormStateInterface;
-use Drupal\entity_browser\Plugin\Field\FieldWidget\EntityReferenceBrowserWidget as BaseEntityReferenceBrowserWidget;
-
-/**
- * Cosmetic enhancements for the Entity Browser entity reference widget.
- */
-class EntityReferenceBrowserWidget extends BaseEntityReferenceBrowserWidget {
```

Comment:
This has been replaced by a hook implementation in lightning_media.module, so that we don't need to bother with plugin hijacking.